### PR TITLE
Chapter 10: Updating, Showing, & Deleting Users

### PR DESCRIPTION
**Pull request by megatimranikram**

### CODE REVIEW

**Comment by megatimranikram** [comment]
ready

File: .gitignore [hunk 1]
```diff
@@ -46,4 +46,4 @@ yarn-debug.log*
 
 config/settings.local.yml
 config/settings/*.local.yml
-config/environments/*.local.yml
+config/environments/*.local.yml
```
Comment (saobv-2223) [inline .gitignore]: add endline, please

File: Gemfile [hunk 2]
```diff
@@ -37,4 +37,4 @@ gem "tzinfo-data", platforms: [:mingw, :mswin, :x64_mingw, :jruby]
 group :development, :test do
   gem "rubocop", "~> 0.74.0", require: false
   gem "rubocop-rails", "~> 2.3.2", require: false
-end
+end
```
Comment (saobv-2223) [inline Gemfile]: add endline, please

File: app/assets/stylesheets/application.css [hunk 3]
```diff
@@ -16,3 +16,4 @@
 
  @import 'bootstrap-sprockets';
  @import 'bootstrap';
+
```
Comment (saobv-2223) [inline app/assets/stylesheets/application.css]: add endline, please

File: app/helpers/application_helper.rb [hunk 4]
```diff
@@ -3,4 +3,4 @@ def full_title page_title
     base_title = "Ruby on Rails Tutorial Sample App"
     page_title.present? ? [page_title, base_title].join(" | ") : base_title
   end
-end
+end
```
Comment (saobv-2223) [inline app/helpers/application_helper.rb]: add endline, please

File: app/helpers/sessions_helper.rb [hunk 5]
```diff
@@ -36,4 +36,17 @@ def forget user
...
+
+  def store_location
+    session[:forwarding_url] = request.original_url if request.get?
+  end
+
+  def redirect_back_or default
+    redirect_to session[:forwarding_url] || default
+    session.delete :forwarding_url
+  end
 end
```
Comment (saobv-2223) [inline app/helpers/sessions_helper.rb]: add endline, please

File: config/application.rb [hunk 6]
```diff
@@ -10,4 +10,4 @@ class Application < Rails::Application
     config.i18n.available_locales = [:en, :ru]
     config.i18n.default_locale = :en
   end
-end
+end
```
Comment (saobv-2223) [inline config/application.rb]: add endline, please

File: config/initializers/config.rb [hunk 7]
```diff
@@ -52,4 +52,4 @@
     #   required(:email).filled(format?: EMAIL_REGEX)
     # end
   
-  end
+  end
```
Comment (saobv-2223) [inline config/initializers/config.rb]: add endline, please

File: db/migrate/20200822064234_add_remember_digest_to_users.rb [hunk 8]
```diff
@@ -0,0 +1,5 @@
+class AddRememberDigestToUsers < ActiveRecord::Migration[6.0]
+  def change
+    add_column :users, :remember_digest, :string
+  end
+end
```
Comment (saobv-2223) [inline db/migrate/20200822064234_add_remember_digest_to_users.rb]: add endline, please

File: db/migrate/20200825125049_add_admin_to_users.rb [hunk 9]
```diff
@@ -0,0 +1,5 @@
+class AddAdminToUsers < ActiveRecord::Migration[6.0]
+  def change
+    add_column :users, :admin, :boolean, default: false
+  end
+end
```
Comment (saobv-2223) [inline db/migrate/20200825125049_add_admin_to_users.rb]: add endline, please

File: db/seeds.rb [hunk 10]
```diff
@@ -1,7 +1,12 @@
...
+password: "11111111",
+password_confirmation: "11111111",
+admin: true)
+99.times do |n|
+  name = Faker::Name.name
+  email = "example-#{n+1}@railstutorial.org"
+  password = "password"
+
+  User.create!(name: name, email: email, password: password, password_confirmation: password, admin: true)
+end
```
Comment (saobv-2223) [inline db/seeds.rb]: add endline, please

**Comment by megatimranikram** [comment]
ready

File: .gitignore (hunk 1 above)
Comment (megatimranikram) [reply .gitignore]: done sir

File: Gemfile (hunk 2 above)
Comment (megatimranikram) [reply Gemfile]: done sir

File: app/assets/stylesheets/application.css (hunk 3 above)
Comment (megatimranikram) [reply app/assets/stylesheets/application.css]: done sir

File: app/helpers/application_helper.rb (hunk 4 above)
Comment (megatimranikram) [reply app/helpers/application_helper.rb]: done sir

File: app/helpers/sessions_helper.rb (hunk 5 above)
Comment (megatimranikram) [reply app/helpers/sessions_helper.rb]: done sir

File: config/application.rb (hunk 6 above)
Comment (megatimranikram) [reply config/application.rb]: done sir

File: config/initializers/config.rb (hunk 7 above)
Comment (megatimranikram) [reply config/initializers/config.rb]: done sir

File: db/migrate/20200822064234_add_remember_digest_to_users.rb (hunk 8 above)
Comment (megatimranikram) [reply db/migrate/20200822064234_add_remember_digest_to_users.rb]: done sir

File: db/migrate/20200825125049_add_admin_to_users.rb (hunk 9 above)
Comment (megatimranikram) [reply db/migrate/20200825125049_add_admin_to_users.rb]: done sir

File: db/seeds.rb (hunk 10 above)
Comment (megatimranikram) [reply db/seeds.rb]: done sir

**Review by saobv-2223 — COMMENTED** [review]
merge

**Pull request merged**